### PR TITLE
Changes the icon of node arranger in GraphEdit & editor

### LINKED ISSUE #51680: Same icon for Minimap & Node arranger in GraphEdit/VisualScript/VisualShader editor
### Godot version

4.0 dev latest

### System information

Any system

### Issue description

Screenshot (302)
Screenshot (301)

We need different icon for node auto-arranger button. Currently, it is using the `.svg` & `.png` file of Minimap icon

### Steps to reproduce

- Open VisualScript/VisualShader editor

**OR**

- Add a `GraphEdit` node.

### PR DESCRIPTION
**Pull request by theoway**
Fixes #51680
 This PR adds new icon for Node Arrangement button in `GraphEdit` and VisualScript/VisualShader editor 

**New icon images below:**
Screenshot (305)
Screenshot (304)

*Bugsquad edit:* Icon designed by @LucyLavend.

### CODE REVIEW

**Review by fire — APPROVED** [review]
Looks good to me.

File: editor/icons/GridLayout.svg [hunk 1]
```diff
@@ -1 +1,25 @@
...
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (akien-mga) [inline editor/icons/GridLayout.svg:1]: Please optimize the SVG with `svgcleaner`: https://docs.godotengine.org/en/stable/development/editor/creating_icons.html#icon-optimization

(Make sure that it still ends with a newline char after running svgcleaner.)

**Comment by theoway** [comment]
@fire I think you need to poke the iOS test. Throwing some weird error.

**Comment by akien-mga** [comment]
Thanks @LucyLavend and @theoway!

**Pull request merged**